subprojects/core/src/main/java/org/gradle/caching/internal/tasks/BuildCacheKeyInputs.java

```diff
@@ -34,6 +34,7 @@
     private final List<HashCode> actionClassLoaderHashes;
     private final ImmutableList<String> actionClassNames;
     private final ImmutableSortedMap<String, HashCode> inputHashes;
+    private ImmutableSortedSet<String> inputPropertyNamesLoadedByUnknownClassLoader;
```

Comment:
Nit: this should either be called `inputPropertiesLoadedByUnknownClassLoader` or `namesOfInputPropertiesLoadedByUnknownClassLoader`.

Reply:
renamed.